.github/workflows/coverage-windows.yml

```diff
@@ -0,0 +1,51 @@
+name: coverage-windows
+
+on:
+  pull_request:
+    paths-ignore:
+      - 'doc/**'
```

Comment:
```suggestion
      - '.github/**'
      - 'doc/**'
```

Reply:
But we would want to run this workflow if the workflow itself (`.github/workflows/coverage-windows.yml`) was modified.